<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="keyLineEdit1">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>610</y>
      <width>125</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>590</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти данные по ключу</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>610</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ключ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultTextEdit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>650</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>64</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>128</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2048</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>321</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Перегенирировать таблицы</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>661</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Значения хэш-функции, вызвавшие коллизию</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1323</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
